aten/src/ATen/native/DistributionTemplates.h

```diff
@@ -64,6 +64,40 @@ at::Tensor& random_impl(at::Tensor& self, at::Generator* generator) {
   return self;
 }
 
+#define show_out_of_bounds_warning(name) \
+  TORCH_WARN(name, " is out of bounds for ", dtype, ". This warning will become an error in version 1.6 release, please fix the code in advance")
+
+static void check_from_to(int64_t from, int64_t to_inc, caffe2::TypeMeta dtype) {
+  const auto scalar_type = typeMetaToScalarType(dtype);
+  if (isFloatingType(scalar_type)) {
+    AT_DISPATCH_FLOATING_TYPES_AND2(at::ScalarType::Half, at::ScalarType::BFloat16, scalar_type, "check_random_fp_bounds", [&] {
+      const auto min = static_cast<double>(std::numeric_limits<scalar_t>::lowest());
+      const auto max = static_cast<double>(std::numeric_limits<scalar_t>::max());
```

Comment:
I think we should use ```min = - std::numeric_limits<scalar_t>::infinity(),  max = std::numeric_limits<scalar_t>::infinity()``` because floating point tensors can be filled with +/- inf, so comparing them with numeric_limits::min/max will produce unexpected results

Reply:
1. `std::numeric_limits<scalar_t>::infinity()` will be casted to `std::numeric_limits<double>::infinity()` and `from < min || from > max` will become just `false` for any type and any value `from`.
2. random accepts only int64_t min and max, why do we need comparison with infinities?